src/main/java/apoc/util/Util.java

```diff
@@ -686,4 +691,99 @@ public static boolean classExists(String className) {
             return null;
         }
     }
+
+    public static Stream<MapResult> runWithRetry( GraphDatabaseService db, Log log, String statement, long retries, List<String> additionalRetryCodes,
+            Map<String,Object> params )
+    {
```

Comment:
please use the regular apoc code format

Reply:
Where could I find the apoc code format scheme?

Reply:
Regular Java Style guide (e.g. IntelliJ / Eclipse defaults)

Reply:
I'll update to the regular style guide as my final step. Want to get all the code completed before changing the aesthetics.